src/app/shared/services/provider/provider.service.ts

```diff
@@ -34,9 +31,8 @@ export class ProviderService {
    * This method create Provider
    * @param Provider
    */
-  createProvider(provider: Provider): Observable<object> {
-    this.isRelease2 = this.store.selectSnapshot<FeaturesList>(MetaDataState.featuresList).release2;
-    return this.isRelease2 ? this.createProviderV2(provider) : this.createProviderV1(provider);
+  createProvider(provider: Provider, isRelease2: boolean): Observable<object> {
+    return isRelease2 ? this.createProviderV2(provider) : this.createProviderV1(provider);
   }
 
   createProviderV1(provider: Provider): Observable<object> {
```

Comment:
Can we find out what the data model is here? for Observable<object>

Reply:
It isn't a part of this pr and isn't scope of ticket. 
I believe, people who made those methods should do that.

Reply:
Unfortunately, according to the specific type of our project, "people who made'"probably are not part of the team anymore/they were students. I am not insisting on this fix, but if you have time and wish - it is very welcome to refactor a piece of code that is not okay:)

Reply:
Based on my previous experience, it is bad idea to made changes which out of scope your ticket. 
But if it's okay to made changes, I will do this changes in this pr.